cmd/diff.go

```diff
@@ -80,14 +80,16 @@ func diffImages(image1Arg, image2Arg string, diffArgs []string) error {
 	for imageArg := range imageMap {
```

Comment:
Could you add a TODO or open a bug to fix up this code, the error handling isn't really correct. You're not making it worse here, bu I want to make sure we don't forget.

Reply:
done.